.github/workflows/ci.yml

```
@@ -29,103 +26,191 @@
+        run: npm run build:analyze
 
-          if [ ! -f "dist/index.js" ]; then
-            echo "❌ Main build file not found"
-            exit 1
-          fi
+      - name: Comment bundle analysis
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)